django/core/management/commands/migrate.py

```diff
@@ -67,14 +67,15 @@ def handle(self, *args, **options):
             warnings.warn(
                 "The 'migrate --list' command is deprecated. Use 'showmigrations' instead.",
                 RemovedInDjango20Warning, stacklevel=2)
+            self.stdout.ending = None
```

Comment:
This will be redundant when https://code.djangoproject.com/ticket/24068 is fixed, right? If yes, it would be good to add a comment about the issue.